2022-02-14 08:20:03,172 INFO    MainThread:17763 [wandb_setup.py:_flush():75] Loading settings from /root/.config/wandb/settings
2022-02-14 08:20:03,172 INFO    MainThread:17763 [wandb_setup.py:_flush():75] Loading settings from /root/data-science-projects/house-price-prediction/Models/tensorflow/wandb/settings
2022-02-14 08:20:03,172 INFO    MainThread:17763 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-14 08:20:03,172 INFO    MainThread:17763 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-02-14 08:20:03,172 INFO    MainThread:17763 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-14 08:20:03,173 INFO    MainThread:17763 [wandb_init.py:_log_setup():386] Logging user logs to /root/data-science-projects/house-price-prediction/Models/tensorflow/wandb/run-20220214_082003-pi18p69c/logs/debug.log
2022-02-14 08:20:03,173 INFO    MainThread:17763 [wandb_init.py:_log_setup():387] Logging internal logs to /root/data-science-projects/house-price-prediction/Models/tensorflow/wandb/run-20220214_082003-pi18p69c/logs/debug-internal.log
2022-02-14 08:20:03,173 INFO    MainThread:17763 [wandb_init.py:_jupyter_setup():336] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f81b54bc700>
2022-02-14 08:20:03,173 INFO    MainThread:17763 [wandb_init.py:init():420] calling init triggers
2022-02-14 08:20:03,173 INFO    MainThread:17763 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {'epochs': 100, 'batch_size': 256}
2022-02-14 08:20:03,174 INFO    MainThread:17763 [wandb_init.py:init():441] re-initializing run, found existing run on stack: 89z7c4nj
2022-02-14 08:20:03,176 INFO    MainThread:17763 [wandb_run.py:finish():1468] finishing run luiscostigan/house-price-prediction/89z7c4nj
2022-02-14 08:20:03,176 INFO    MainThread:17763 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-02-14 08:20:03,178 INFO    MainThread:17763 [wandb_run.py:_restore():1769] restore
2022-02-14 08:20:03,278 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1025
  total_bytes: 1025
}

2022-02-14 08:20:03,685 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1025
  total_bytes: 1025
}

2022-02-14 08:20:03,788 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1025
  total_bytes: 4291
}

2022-02-14 08:20:03,891 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1025
  total_bytes: 4291
}

2022-02-14 08:20:03,993 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1025
  total_bytes: 4291
}

2022-02-14 08:20:04,096 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1025
  total_bytes: 4291
}

2022-02-14 08:20:04,199 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4291
  total_bytes: 4291
}

2022-02-14 08:20:04,302 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4291
  total_bytes: 4291
}

2022-02-14 08:20:04,404 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4291
  total_bytes: 4291
}

2022-02-14 08:20:04,506 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4291
  total_bytes: 4291
}

2022-02-14 08:20:04,608 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4291
  total_bytes: 4291
}

2022-02-14 08:20:05,361 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4291
  total_bytes: 4291
}

2022-02-14 08:20:06,171 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4291
  total_bytes: 4291
}
local_info {
}

2022-02-14 08:20:07,703 INFO    MainThread:17763 [wandb_run.py:_append_files():2194] logging synced files
2022-02-14 08:20:07,707 INFO    MainThread:17763 [wandb_init.py:init():471] starting backend
2022-02-14 08:20:07,707 INFO    MainThread:17763 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-14 08:20:07,709 INFO    MainThread:17763 [backend.py:ensure_launched():219] starting backend process...
2022-02-14 08:20:07,723 INFO    MainThread:17763 [backend.py:ensure_launched():224] started backend process with pid: 406
2022-02-14 08:20:07,725 INFO    MainThread:17763 [wandb_init.py:init():480] backend started and connected
2022-02-14 08:20:07,749 INFO    MainThread:17763 [wandb_run.py:_label_probe_notebook():921] probe notebook
2022-02-14 08:20:07,749 INFO    MainThread:17763 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-14 08:20:07,749 INFO    MainThread:17763 [wandb_init.py:init():550] updated telemetry
2022-02-14 08:20:07,792 INFO    MainThread:17763 [wandb_init.py:init():581] communicating current version
2022-02-14 08:20:08,623 INFO    MainThread:17763 [wandb_init.py:init():586] got version response 
2022-02-14 08:20:08,623 INFO    MainThread:17763 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-02-14 08:20:09,020 INFO    MainThread:17763 [wandb_init.py:init():624] starting run threads in backend
2022-02-14 08:20:10,613 INFO    MainThread:17763 [wandb_run.py:_console_start():1827] atexit reg
2022-02-14 08:20:10,614 INFO    MainThread:17763 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2022-02-14 08:20:10,614 INFO    MainThread:17763 [wandb_run.py:_redirect():1738] Wrapping output streams.
2022-02-14 08:20:10,615 INFO    MainThread:17763 [wandb_run.py:_redirect():1762] Redirects installed.
2022-02-14 08:20:10,615 INFO    MainThread:17763 [wandb_init.py:init():651] run started, returning control to user process
2022-02-14 08:20:12,439 INFO    MainThread:17763 [wandb_init.py:_pause_backend():299] pausing backend
2022-02-14 08:20:12,439 INFO    MainThread:17763 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-14 08:20:31,930 INFO    MainThread:17763 [wandb_init.py:_resume_backend():308] resuming backend
2022-02-14 08:20:31,939 INFO    MainThread:17763 [wandb_setup.py:_flush():75] Loading settings from /root/.config/wandb/settings
2022-02-14 08:20:31,939 INFO    MainThread:17763 [wandb_setup.py:_flush():75] Loading settings from /root/data-science-projects/house-price-prediction/Models/tensorflow/wandb/settings
2022-02-14 08:20:31,939 INFO    MainThread:17763 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-14 08:20:31,940 INFO    MainThread:17763 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-02-14 08:20:31,940 INFO    MainThread:17763 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-14 08:20:31,940 INFO    MainThread:17763 [wandb_init.py:_log_setup():386] Logging user logs to /root/data-science-projects/house-price-prediction/Models/tensorflow/wandb/run-20220214_082031-1bp04nnh/logs/debug.log
2022-02-14 08:20:31,940 INFO    MainThread:17763 [wandb_init.py:_log_setup():387] Logging internal logs to /root/data-science-projects/house-price-prediction/Models/tensorflow/wandb/run-20220214_082031-1bp04nnh/logs/debug-internal.log
2022-02-14 08:20:31,940 INFO    MainThread:17763 [wandb_init.py:init():420] calling init triggers
2022-02-14 08:20:31,940 INFO    MainThread:17763 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {'epochs': 100, 'batch_size': 256}
2022-02-14 08:20:31,940 INFO    MainThread:17763 [wandb_init.py:init():441] re-initializing run, found existing run on stack: pi18p69c
2022-02-14 08:20:31,942 INFO    MainThread:17763 [wandb_run.py:finish():1468] finishing run luiscostigan/house-price-prediction/pi18p69c
2022-02-14 08:20:31,943 INFO    MainThread:17763 [jupyter.py:save_history():429] not saving jupyter history
2022-02-14 08:20:31,943 INFO    MainThread:17763 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-14 08:20:31,943 INFO    MainThread:17763 [wandb_init.py:_jupyter_teardown():318] cleaning up jupyter logic
2022-02-14 08:20:31,943 INFO    MainThread:17763 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-02-14 08:20:31,946 INFO    MainThread:17763 [wandb_run.py:_restore():1769] restore
2022-02-14 08:20:32,153 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1025
  total_bytes: 1025
}

2022-02-14 08:20:32,583 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1025
  total_bytes: 1025
}

2022-02-14 08:20:33,050 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1025
  total_bytes: 4264
}

2022-02-14 08:20:33,153 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1025
  total_bytes: 4290
}

2022-02-14 08:20:33,255 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1025
  total_bytes: 4290
}

2022-02-14 08:20:33,358 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1025
  total_bytes: 4290
}

2022-02-14 08:20:33,461 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4290
  total_bytes: 4290
}

2022-02-14 08:20:33,564 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4290
  total_bytes: 4290
}

2022-02-14 08:20:33,667 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4290
  total_bytes: 4290
}

2022-02-14 08:20:33,769 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4290
  total_bytes: 4290
}

2022-02-14 08:20:33,872 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4290
  total_bytes: 4290
}

2022-02-14 08:20:33,973 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4290
  total_bytes: 4290
}

2022-02-14 08:20:34,593 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4290
  total_bytes: 4290
}

2022-02-14 08:20:35,173 INFO    MainThread:17763 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4290
  total_bytes: 4290
}
local_info {
}

2022-02-14 08:20:36,706 INFO    MainThread:17763 [wandb_run.py:_append_files():2194] logging synced files
